<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotParamTheta</class>
 <widget class="QWidget" name="PlotParamTheta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>83</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>17</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="changeColorMap">
       <property name="text">
        <string>Change transfer function</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icons/palette.png</normaloff>:/icons/palette.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Min. move</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="moveValue">
       <property name="suffix">
        <string/>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="moveUnit">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>nm</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Âµm</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mm</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>m</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
